code/go/0chain.net/blobber/logging.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/config"
+	"github.com/0chain/blobber/code/go/0chain.net/core/logging"
+	"github.com/0chain/gosdk/zcncore"
+)
+
+func setupLogging() {
+	fmt.Print("[3/10] init logging")
+
+	if config.Development() {
+		logging.InitLogging("development", logDir, "0chainBlobber.log")
+	} else {
+		logging.InitLogging("production", logDir, "0chainBlobber.log")
+	}
+
+	zcncore.SetLogFile(logDir+"/0chainBlobber.log", false)
+	zcncore.SetLogLevel(3)
+	fmt.Print("		[OK]\n")
+}
```

Comment:
Do we need this? It is actually a very trivial function, what you suggest?

Reply:
Yes. It makes `main` clean.
- flags
- config
- logging
- minio
- node
- server chain
- database
- filestore
- tests
- china
- server